js/Active-Links.js

```diff
@@ -1,26 +1,35 @@
-/* eslint-disable func-names */
-$(document).ready(() => {
-  $('.navbar-nav li a').click(function () {
-    $('.navbar-nav li a').parent().removeClass('active');
-    $(this).parent().addClass('active');
-  });
+(function () {
+  const $navbarLinksSelector = $('.navbar-nav li a'),
+    $window = $(window);
+
+  function onLinkClick() {
+    const $this = $(this);
```

Comment:
what is `$this`  means ? what element is it ?

Reply:
So, everytime when I'm writing `$(this)`, I'm creating new jQuery object.
Instead of this, I defined variable `$this` once, wich has link to new created jQuery object.